src/zenhub.py

```diff
@@ -219,66 +128,29 @@ def _update_issue_pipeline(self):
         Issue pipeline name must be valid. By default issues are inserted at the top of the list in the pipeline."""
 
         if self.pipeline in self.repo.pipeline_ids:
-            logger.info(f'Changing the current value of pipeline to {self.pipeline}')
-
             json_dict = {'pipeline_id': self.repo.pipeline_ids[self.pipeline], 'position': 'top'}
+            self.repo.api_call(requests.post, f'{self.repo.id}/issues/{self.github_key}/moves', json=json_dict)
 
-            response = requests.post(f'{self.repo.url}{self.repo.id}/issues/{self.github_key}/moves',
-                              headers=self.repo.headers, json=json_dict)
-
-            if response.status_code == 200:
-                logger.info(f'Success. {self.github_key} was moved to {self.pipeline}')
-            else:
-                logger.debug(
-                    f'{response.status_code} error occurred when updating issue {self.github_key} pipeline: {response.text}')
-                raise RuntimeError(
-                    f'{response.status_code} error occurred when updating issue {self.github_key} pipeline: {response.text}')
         else:
-            print("not a valid pipeline")
-            logger.debug(f'{self.pipeline} is not a valid pipeline.')
+            print(f'Cannot update issue {self.github_key} pipeline: not a valid pipeline')
```

Comment:
Why is this printed? Logging would be preferred.

Reply:
Changed to logging